installer/webapp/pom.xml

```diff
@@ -155,7 +155,24 @@
             <type>war</type>
         </dependency -->
 
+        <!-- https://mvnrepository.com/artifact/javax.servlet/javax.servlet-api -->
         <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <version>4.0.1</version>
```

Comment:
If we aren't seeing issues with 8.5 then I would think it would be fine.  The original 8 is now EOF.

Reply:
Sorry, I wasn't referring to 8.0.x at all, I was referring to 8.5.x. The [changelog](https://ci.apache.org/projects/tomcat/tomcat85/docs/changelog.html) for 8.5 mentions the possible removal of the "Servlet 4.0 Preview API" which I can't find any good information on. I'm not clear on the implications of upgrading the servlet API to 4 when Tomcat 8.5 doesn't (apparently) support it. Perhaps I'm not seeing any issues because VIVO doesn't contain any functionality introduced in Servlet 4.0, and the servlet classes are backwards compatible? 

All this confusion aside, if the result is Tomcat 9 is a requirement for the next version of VIVO that's fine with me as long as the issues @awoods ran into using 9 are sussed out.

Reply:
I thought this conversation seemed familiar, hah. See https://github.com/vivo-project/Vitro/pull/109#pullrequestreview-191080148 for previous discussion.